src/entities/RealmProvider/utils.ts

```diff
@@ -1,47 +1,58 @@
 import Time from "decentraland-gatsby/dist/utils/date/Time"
 import env from "decentraland-gatsby/dist/utils/env"
 import fetch, { RequestInit } from "node-fetch"
+import deepmerge from "deepmerge"
```

Comment:
what do you think about using [radash](https://radash-docs.vercel.app/docs/array/merge) that's in decentraland gatsby?